config_converter/config_mapper/config_mapper.py

```diff
@@ -86,7 +90,7 @@ def _convert_in_tail(d: config_pb2.Directive) -> dict:
         'expression'
     ] + [f'format{i}' for i in range(1, 21)]
     for p in d.params:
-        if p.name == '@type' or p.name == 'tag':
+        if p.name in ['@type', 'tag', '@log_level']:
```

Comment:
It really doesn't matter here, but just note that in Python 3 you could create a set instead of a list:

    {'@type', 'tag', '@log_level}

Reply:
Oh right - I can fix that